src/main/helper/ApiFormatter.ts

```diff
@@ -481,7 +481,8 @@ export const setDocumentValues = (
 };
 
 export const getDocId = (url: string): string => {
-  return url.substring(url.lastIndexOf('/') + 1, url.length);
+  const x = url?.substring(url.lastIndexOf('/') + 1, url.length);
```

Comment:
return x right away

Reply:
Why this change?

Reply:
Mistakenly left in while debugging. Removed.